### ISSUE TITLE
Compilation error while following instructions

### ISSUE THREAD

**Issue by webdev** [mesos/mesos-go#51]
I'm getting an error while trying to follow build instructions.

```
root@441a0c5a33df:/go/mesos-go/examples# go get github.com/golang/glog github.com/mesos/mesos-go/mesosproto github.com/mesos/mesos-go/mesosutil github.com/mesos/mesos-go/scheduler
# github.com/mesos/mesos-go/mesosproto
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:661: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:720: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:737: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:754: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:771: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:789: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:806: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:833: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:860: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:970: undefined: proto.ErrWrongType
/go/src/github.com/mesos/mesos-go/mesosproto/log.pb.go:970: too many errors
```

Any ideas?

**Comment by diptanu** [comment]
@vladimirvivien @nqn goprotobuf and gogoprotobuf removed proto.ErrWrongType.
We need to regenerate the proto files with a new version of goprotobuf or gogoprotobuf.

**Comment by diptanu** [comment]
@vladimirvivien @nqn I have submitted #54 to fix the problem. Can you guys take a look please?

**Comment by diptanu** [comment]
@vladimirvivien @nqn Just tested that the issue is fixed after #54 was merged. You can close this issue.

**Comment by sangpire** [comment]
@diptanu I'm getting an error while trying to [build instructions](https://github.com/mesos/mesos-go#build-instructions) too.

```
$ go build ./...
mesosproto/log.pb.go:29:8: cannot find package "github.com/gogo/protobuf/proto" in any of:
	/usr/local/Cellar/go/1.4/libexec/src/github.com/gogo/protobuf/proto (from $GOROOT)
	/Users/sangpire/playground/go/src/github.com/gogo/protobuf/proto (from $GOPATH)
```

It looks like forget Godeps.json Update.

**Comment by sangpire** [comment]
Hi @diptanu
I'm getting an error too.

```
$ godep restore
$ go build ./...
mesosproto/log.pb.go:29:8: cannot find package "github.com/gogo/protobuf/proto" in any of:
	/usr/local/Cellar/go/1.4/libexec/src/github.com/gogo/protobuf/proto (from $GOROOT)
	/Users/sangpire/playground/go/src/github.com/gogo/protobuf/proto (from $GOPATH)
```

It looks like forget updating Godeps.json for _github.com/gogo/protobuf/proto_

**Comment by diptanu** [comment]
@sangpire Yes, sorry I didn't update the Godeps.json. I will do that and send a PR

**Comment by diptanu** [comment]
@sangpire @nqn Just submitted another PR - #56 This should compile the code with godep too.

**Comment by vladimirvivien** [comment]
Let me take a look. Thanks for migrating to the new package github location.

**Comment by vladimirvivien** [comment]
This was fixed with PR #56
